packages/react-slider/src/common/isConformant.ts

```diff
@@ -1,8 +1,9 @@
 import { isConformant as baseIsConformant, IsConformantOptions } from '@fluentui/react-conformance';
 
-export function isConformant(testInfo: IsConformantOptions) {
+export function isConformant(testInfo: Omit<IsConformantOptions, 'componentPath'>) {
```

Comment:
this typing change looks incorrect to me. why is this here?

Reply:
same for other places

Reply:
Because componentPath is now being calculated automatically based on the test filename (see below)

Reply:
then shouldn't we update `IsConformantOptions` typing instead?

Reply:
No, the `componentPath` is a project-specific detail which each project's `isConformant` wrapper may be able to provide for its own components, but the `react-conformance` package can't provide generically.